src/inmanta/util/dict_path.py

```diff
@@ -223,6 +224,17 @@ def get_elements(self, container: object) -> list[object]:
         :param container: the container to search in
         """
 
+    @abc.abstractmethod
+    def get_paths(self, container: object) -> list["DictPath"]:
+        """
+        Get all the dict paths that this wild dict path contains, in the given
+        container.  That is, the set of dict path that would resolve the same
+        set of elements as this wild dict path, when searching in the given
+        container.
+
+        :param container: the container to search in
+        """
```

Comment:
This is probably not clear enough, but I am really struggling with the explanation and the naming of such a feature.  Any suggestion?